### DISCUSSION TITLE
Can't test with Python 3.10-beta.3

### DISCUSSION THREAD

**Discussion by thedrow** [ymyzk/tox-gh-actions#168] [Q&A] [answered]
It seems like tox-gh-actions has a problem parsing these type of Python versions.
See [build log](https://github.com/celery/celery/pull/6807/checks?check_run_id=2924732834).

**Comment by ymyzk** [comment, marked as answer]
This format `3.10-beta.3` is not supported by tox-gh-actions. Please simply specify `3.10` instead.

```yaml
[gh-actions]
python =
    3.6: 3.6-unit
    3.7: 3.7-unit
    3.8: 3.8-unit
    3.9: 3.9-unit
    3.10: 3.10-unit
    pypy3: pypy3-unit
```

**Comment by thedrow** [comment]
That works. Thanks!
